### PR TITLE
Torchscript test for DistilBERT

### PR DESCRIPTION
**Pull request by LysandreJik**
cc @HamidShojanazeri, test for https://github.com/huggingface/transformers/pull/12290

### CODE REVIEW

File: tests/test_modeling_distilbert.py [hunk 1]
```diff
@@ -252,6 +252,29 @@ def test_model_from_pretrained(self):
...
+            config.torchscript = True
+            model = model_class(config=config)
+
+            inputs_dict = self._prepare_for_class(inputs_dict, model_class)
+            traced_model = torch.jit.trace(
+                model, (inputs_dict["input_ids"].to("cpu"), inputs_dict["attention_mask"].to("cpu"))
+            )
+
+            with tempfile.TemporaryDirectory() as tmp:
+                torch.jit.save(traced_model, os.path.join(tmp, "bert.pt"))
```
Comment (HamidShojanazeri) [inline tests/test_modeling_distilbert.py:274]: nit : maybe we could change "bert.pt" to "traced_model.pt"

**Review by HamidShojanazeri — APPROVED** [review]
Thanks @LysandreJik.

File: tests/test_modeling_distilbert.py (hunk 1 above)
Comment (LysandreJik) [inline tests/test_modeling_distilbert.py:274]:
```suggestion
                torch.jit.save(traced_model, os.path.join(tmp, "traced_model.pt"))
```

Comment (LysandreJik) [reply tests/test_modeling_distilbert.py:274]: Good point!

**Pull request merged**